### PR TITLE
Added the ability to add content to the switch

### PR DESCRIPTION
**Pull request by mbejda**
Added the ability to add content to the switch so its customizable. 

You can add 
- web fonts
- images
- animations
- custom html

This closes feature request https://github.com/webcat12345/ngx-ui-switch/issues/311 and doesn't introduce any breaking changes.

### CODE REVIEW

File: src/app/demo/demo.component.html [hunk 1]
```diff
@@ -108,5 +108,38 @@ <h2>Custom labels</h2>
...
+    <div class="row">
+      <div class="col-12">
+        <h2>Custom Switch Content</h2>
+        <p class="custom-icons">
+          <ui-switch uncheckedLabel="off"
+                     checkedLabel="on"
+                     size="small">
+            <img
+              style="height:10px; width:10px; margin-left:30%;margin-top:1%;"
+              src="/assets/icons/feather/activity.svg"/>
```
Comment (cmckni3) [inline src/app/demo/demo.component.html]: This image won't be pulled from the assets correctly when running `ng build --prod`.

Use `assets/icons/feather/activity.svg` instead.

Comment (cmckni3) [reply src/app/demo/demo.component.html]: Personally, I would recommend adding `font-awesome` to the top level `package.json` and `src/styles.scss`

**Comment by cmckni3** [comment]
Thanks! Looks great but I would rather use font awesome or another font icon in the demo.

I'm not opposed to using `feather-icons` as a [package](https://github.com/feathericons/feather#client-side-javascript) in the root level `package.json` though.

**Comment by mbejda** [comment]
Rebased incorrectly. New PR with changes is here. https://github.com/webcat12345/ngx-ui-switch/pull/313

**Pull request closed without merging**